README.md

```diff
@@ -192,6 +174,20 @@ jobs:
         run: echo "run steps in container"
 ```
 
+#### Login to Amazon ECR Private while masking the password:
+
+> [!WARNING]
+> Setting `mask-password` to `'true'` will prevent the Docker password output from being shared between separate jobs.
+> For more information, see the [Docker Credentials](#docker-credentials) section below.
```

Comment:
Should we reverse this warning? On the example of how to "Use the image as a service", shouldn't it say "WARNING: setting mask-password to `false` will cause the registry password to be sent to GitHub actions and show up in debug logs"?

Reply:
Added